### DISCUSSION TITLE
Adding Postman Collection for API Endpoint Testing

### DISCUSSION THREAD

**Discussion by poksyy** [poksyy/hospital-project-backend#57] [Show and tell]
A Postman collection, [Nurse_API_Tests.json](https://github.com/user-attachments/files/17750926/Nurse_API_Tests.json), is now available for testing all API endpoints. This file contains predefined requests for the main API functions (login, retrieval by ID and name, creation, update, and deletion). The collection allows contributors to quickly verify API responses and check for expected behavior.

### **To use it, download the .json file and import it into Postman:**

- Open Postman, go to File > Import.
- Select the .json file, and Postman will load the collection with all configured requests.
- Each request can be run individually to test the application endpoints.

#56 

Feel free to provide feedback or suggest updates if you encounter issues or have ideas for additional test cases.

**Comment by krschan** [comment]
https://youtu.be/kI5buEVUugw?si=RNmqObaH6KZx6F0a
